build.gradle.kts

```
@@ -79,19 +63,12 @@ dependencies {
     implementation("org.ow2.asm:asm-tree:9.2")
     implementation("com.googlecode.java-diff-utils:diffutils:1.3.0")
     compileOnly("org.jetbrains.kotlin.multiplatform:org.jetbrains.kotlin.multiplatform.gradle.plugin:1.8.10")
-//    compileOnly("com.android.tools.build:gradle:${androidGradlePluginVersion}")
+    compileOnly("com.android.tools.build:gradle:${androidGradlePluginVersion}")
```

This one is worth keeping commented out until #108 is supported

---

```suggestion
    //compileOnly("com.android.tools.build:gradle:${androidGradlePluginVersion}")
```

good point, I'm not sure why I uncommented it.